dts/bindings/sensor/st,lsm6dso-common.yaml

```diff
@@ -121,6 +121,18 @@ properties:
     description: |
       Enable FIFO buffer
 
+  raw-readings-en:
+    type: boolean
+    description: |
+      Send out raw register values of the readings instead of converted unit values.
+      Only relevant if FIFO mode active
+
+  unit-readings-en:
+    type: boolean
+    description: |
+      Send out normal unit converted values of the readings instead of converted unit values.
```

Comment:
I don't understand this line, and my hunch is that it's not due to the my lack of domain knowledge.

Reply:
that's jsut plain rubbish I wrote here. changed!